src/WebJobs.Extensions.Twilio/Config/TwilioJobHostConfigurationExtensions.cs

```diff
@@ -0,0 +1,113 @@
+        /// <param name="twilioSMSConfig">The <see cref="TwilioSMSConfiguration"/> to use.</param>
+        public static void UseTwilioSMS(this JobHostConfiguration config, TwilioSMSConfiguration twilioSMSConfig = null)
+        {
+            if (config == null)
+            {
+                throw new ArgumentNullException("config");
+            }
+
+            if (twilioSMSConfig == null)
+            {
+                twilioSMSConfig = new TwilioSMSConfiguration();
+            }
+
+            config.RegisterExtensionConfigProvider(new TwilioSMSConfig(twilioSMSConfig));
+        }
+
+        internal class TwilioSMSConfig : IExtensionConfigProvider
+        {
+            private TwilioSMSConfiguration _twilioConfig;
+            private TwilioRestClient _twilioClient;
+
+            public TwilioSMSConfig(TwilioSMSConfiguration twilioConfig)
+            {
+                _twilioConfig = twilioConfig;
+            }
+
+            public void Initialize(ExtensionConfigContext context)
+            {
+                if (context == null)
+                {
+                    throw new ArgumentNullException("context");
+                }
+
+                if (string.IsNullOrEmpty(_twilioConfig.AccountSid))
+                {
+                    ThrowMissingSettingException("AccountSID", TwilioSMSConfiguration.AzureWebJobsTwilioAccountSidKeyName, "AccountSID");
+                }
+
+                if (string.IsNullOrEmpty(_twilioConfig.AuthToken))
+                {
+                    ThrowMissingSettingException("AuthToken", TwilioSMSConfiguration.AzureWebJobsTwilioAccountAuthTokenName, "AuthToken");
+                }
+
+                _twilioClient = new TwilioRestClient(_twilioConfig.AccountSid, _twilioConfig.AuthToken);
+
+                IConverterManager converterManager = context.Config.GetService<IConverterManager>();
+                converterManager.AddConverter<JObject, SMSMessage>(CreateSmsMessage);
+
+                INameResolver nameResolver = context.Config.GetService<INameResolver>();
+                BindingFactory factory = new BindingFactory(nameResolver, converterManager);
+                IBindingProvider outputProvider = factory.BindToAsyncCollector<TwilioSMSAttribute, SMSMessage>((attr) =>
+                {
+                    return new TwilioSMSMessageAsyncCollector(_twilioConfig, attr, _twilioClient);
+                });
+
+                IExtensionRegistry extensions = context.Config.GetService<IExtensionRegistry>();
+                extensions.RegisterBindingRules<TwilioSMSAttribute>(outputProvider);
+            }
+
+            private SMSMessage CreateSmsMessage(JObject messageJson)
+            {
+                var message = new SMSMessage();
+                message.From = GetPropertyValue(messageJson, "from");
+                message.To = GetPropertyValue(messageJson, "to");
+                message.Body = GetPropertyValue(messageJson, "body"); 
```

Comment:
Isn't there a MediaUrl property as well for including pictures/video? If so, might be good to include